tests/test_theta_limiter.py

```diff
@@ -73,17 +71,15 @@ def setup_theta_limiter(dirname):
     u0.project(gradperp(psi0))
     theta0.interpolate(theta_b + theta_pert)
 
-    state.initialise([('u', u0), ('rho', rho0), ('theta', theta0)])
+    state.initialise([('u', u0), ('theta', theta0)])
     state.set_reference_profiles([('theta', theta_b)])
 
     # set up advection schemes
-    rhoeqn = EmbeddedDGAdvection(state, Vr, equation_form="continuity")
     thetaeqn = EmbeddedDGAdvection(state, Vt, equation_form="advective")
 
     # build advection dictionary
     advected_fields = []
     advected_fields.append(('u', NoAdvection(state, u0, None)))
-    advected_fields.append(('rho', SSPRK3(state, rho0, rhoeqn)))
     advected_fields.append(('theta', SSPRK3(state, theta0, thetaeqn, limiter=ThetaLimiter(thetaeqn.space))))
```

Comment:
I wonder if it is clearer to give the limiter the equation that it is applied to?